src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx

```diff
@@ -47,6 +47,27 @@ class ProjectKeyCredentials extends React.Component {
 
     return (
       <React.Fragment>
+        {showDsn && (
+          <Field
+            label={t('DSN (Legacy)')}
+            help={t(
+              'This DSN includes the secret which is no longer required by Sentry or newer versions of SDKs.'
+            )}
+            inline={false}
+            flexibleControlStateSize
+          >
+            <TextCopyInput>
+              {getDynamicText({
+                value: data.dsn.secret,
+                fixed: data.dsn.secret.replace(
+                  new RegExp(`\/${projectId}$`),
```

Comment:
You missed a merge here

Reply:
Sorry, not really sure what you mean by that 🙃